lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'menu'
+
+class Orders
+
+attr_reader :order, :message, :text
+
+  def initialize
+    @order = []
+    @text = Text.new
+  end
+
+  def add_menu_item_to_order(item, number=1)
+    @order << [item, number, number *item.price]
+  end
+
+  def total_dishes
+  total_dishes = []
+    order.each do |item|
+      total_dishes <<item[1]
+    end
+  total_dishes.inject(:+)
+  end
+
+  def show_order
+    @order.each do |item|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `item`. You can omit the argument if you don't care about it.